roman_numeral_converter_spec.rb

```diff
@@ -0,0 +1,70 @@
+require_relative "./roman_numeral_converter.rb"
+
+describe "Roman Numeral Converter" do
+  context "roman numeral to arabic conversion" do
+    before { @converter = RomanArabicConverter.new }
+
+    context "roman numeral is valid" do
+      context "roman numeral is one character" do
+        specify { expect(@converter.convert_to_arabic("I")).to eq(1) }
+        specify { expect(@converter.convert_to_arabic("V")).to eq(5) }
+        specify { expect(@converter.convert_to_arabic("X")).to eq(10) }
+        specify { expect(@converter.convert_to_arabic("L")).to eq(50) }
+        specify { expect(@converter.convert_to_arabic("C")).to eq(100) }
```

Comment:
:turtle: NAB: If you use aggregate failures as specified in another comment you can optionally be more declarative with the specs like this:

````ruby
let(:input) { %w(I II IV) }  #=> ["I", "II", "IV"]
let(:results) { [1, 2, 4] }  #=> [1, 2, 4]

specify do
  Hash[input.zip(results)].each do |input, result|  # Hash[input.zip(results)] => { "I" => 1, "II" => 2, "IV" => 4}
    expect converter.convert_to_arabic(input).to eq result
  end
end

````